scripts/Deploy-MSBuild.ps1

```diff
@@ -50,6 +50,20 @@ $BackupFolder = New-Item (Join-Path $destination -ChildPath "Backup-$(Get-Date -
 Write-Verbose "Copying $configuration MSBuild to $destination"
 Write-Host "Existing MSBuild assemblies backed up to $BackupFolder"
 
+if ($runtime -eq "Detect") {
+    if ($destination -like "*dotnet*sdk*") {
+        $runtime = "Core"
+        Write-Host "Detected path that looks like an sdk. Writing .NET Core assemblies."
+    }
+    else {
+        $runtime = "Desktop"
+        Write-Host "Detected path that does not look like an sdk. Writing .NET Framework assemblies."
+    }
+}
+else if ($runtime -eq "Full") {
```

Comment:
Is it possible to get here with the `ValidateSet` from above?

Reply:
I didn't realize the ValidateSet applied to the line below it. I'd prefer to add both Detect and Full to the "valid" set. It hadn't been a problem 'til now just because Desktop is the default; if I have some reason to want to override it, I'll put in what, to me, is the logical opposite of Core, namely Full, and be confused when it fails.